lib/src/utils.test.ts

```diff
@@ -5,7 +5,7 @@ describe('utils', () => {
   describe('pathToUrlString', () => {
     it('encode relative path like `pathToFileURL`', () => {
       const baseURL = pathToFileURL('').toString();
-      for (let i = 0; i < 128; i++) {
+      for (let i = 33; i < 128; i++) {
```

Comment:
@nex3 Node versions 18.19.0 and 20.6.0 introduced a breaking change where `pathToFileURL` now trims trailing whitespace from the provided path. This is a workaround (skipping test for whitespace characters) while we await feedback on the [related issue](https://github.com/nodejs/node/issues/51167) I filed with Node.

Another option would be to adjust our implementation of `pathToUrlString` to similarly trim leading and trailing whitespace.

Reply:
Nice work tracking this down! I think this workaround is fine for now, but can you add a comment linking to the tracking issue?

Reply:
Comment added in https://github.com/oddbird/embedded-host-node/commit/f9a28ed3debb6655e1e42a94afef902678b5d68f